crates/fuel-core/src/service/adapters/graphql_api.rs

```diff
@@ -145,3 +148,18 @@ impl worker::BlockImporter for BlockImporterAdapter {
         )
     }
 }
+
+impl AtomicView for Database {
+    type View = Database;
+
+    fn view_at(&self, _: BlockHeight) -> StorageResult<Self::View> {
+        unimplemented!(
+            "Unimplemented until of the https://github.com/FuelLabs/fuel-core/issues/451"
+        )
+    }
+
+    fn latest_view(&self) -> Self::View {
+        // TODO: https://github.com/FuelLabs/fuel-core/issues/1581
```

Comment:
I feel like we should fix this TODO as well, now that the linked issue is no longer blocked. I'm fine with a follow-up PR as well.